idseq_pipeline/commands/host_filtering_functions.py

```diff
@@ -227,12 +227,17 @@ def run_star_part(output_dir, genome_dir, fastq_files, count_genes=False):
                            '--genomeDir', genome_dir,
                            '--readFilesIn', " ".join(fastq_files)]
     if fastq_files[0][-3:] == '.gz':
-        star_command_params += ['--readFilesCommand', 'zcat']
+        if "fasta" in fastq_files[0]:
```

Comment:
these checks are strictly less precise than the old code - e.g. a file named "thisisnotafastafile.fastq.gz" would meet this condition. also "thisisnotafastqfile.fa.gz" would be considered a fastq file

Reply:
STAR takes both fasta/fastq, it would figure it out itself we are just checking if it's gzipped or not.

Reply:
i know - it's a minor thing but 'if "fasta" in...' just checks if 'fasta' is anywhere in the filename, while the old code looked for it as an extension